ubiquity/dell-bootstrap.py

```diff
@@ -535,9 +535,21 @@ def fixup_recovery_devices(self):
             partition = item.get_partition()
             if loop or \
                partition or \
-               not block or \
-               not block.get_cached_property("HintPartitionable").get_boolean() or \
-               block.get_cached_property("ReadOnly").get_boolean():
+               not block:
```

Comment:
This lost two hints.  Do those cause failures here?  I think specifically `ReadOnly` is useful to maintain.

Reply:
Yes, I will update the code to hold "ReadOnly" value check